swift_browser_ui_frontend/src/components/ObjectTable.vue

```diff
@@ -430,17 +442,19 @@ export default {
         });
       }
     },
+    removeToast: function() {
+      this.enableDownload();
+      document.querySelector("#largeDownload-toasts")
+        .removeToast("largeDownload");
+    },
     confirmDownload: function () {
-      // Snackbar for enabling large downloads for the duration of the
-      // session
-      this.$buefy.snackbar.open({
-        duration: 5000,
-        message: this.$t("message.largeDownMessage"),
-        type: "is-success",
-        position: "is-top",
-        actionText: this.$t("message.largeDownAction"),
-        onAction: this.enableDownload,
-      });
+      document.querySelector("#largeDownload-toasts").addToast(
+        { type: "info",
```

Comment:
the snackbar was success and this became info, is that a design decision or something got lost when moving to it ?

Reply:
For this one with the text phrased as negative
```No large (> 1GiB) downloads enabled. Click to enable them for the duration of the session.```
"success" didn't seem most appropriate. But hopefully we'll get Aino to review all the UI with all the updated elements in the migration :)

Reply:
maybe warning, but I can live with `info` and Aino making the decision on it.